wire-ios-sync-engine/Tests/Source/Synchronization/Transcoders/ZMMissingUpdateEventsTranscoderTests.m

```diff
@@ -401,6 +407,8 @@ - (void)testThatTheLastUpdateEventIDIsReadFromTheManagedObjectContext
     WaitForAllGroupsToBeEmpty(0.5);
 
     // when
+    LastEventIDRepository *lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.userIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
same

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d